### DISCUSSION TITLE
Offer TailwindUI snippets in twin.macro

### DISCUSSION THREAD

**Discussion by corysimmons** [tailwindlabs/tailwindcss#8507] [Ideas]
I want to copy/paste TailwindUI, but also want to use twin.macro pretty badly... Right now I have to copy/paste/convert and it's a really slow process.

If it's possible to have a converter offer twin.macro as an option right here

![image](https://user-images.githubusercontent.com/224636/171761781-f9b795be-359d-486f-a9bf-7d87d0db9823.png)

I would be super excited.

**Comment by corysimmons** [comment]
tagging @ben-rogerson in case he can help

**Reply by ben-rogerson** [reply]
It'll be quite a manual conversion process unless you use the config option: `{ includeClassNames: true }` which allows twin to visit className attributes/props.
